staging/publishing/rules.yaml

```diff
@@ -1490,6 +1490,14 @@ rules:
       dirs:
       - staging/src/k8s.io/cri-api
   library: true
+- destination: cri-client
+  branches:
+  - name: master
+    source:
+      branch: master
+      dirs:
+      - staging/src/k8s.io/cri-client
+  library: true
```

Comment:
This it tricky, either `pull-publishing-bot-validate` fails with:

```
I0430 08:52:21.043970    1845 rules.go:109] loading rules file : /home/prow/go/src/k8s.io/kubernetes/staging/publishing/rules.yaml
I0430 08:52:21.0543[6](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/123797/pull-publishing-bot-validate/1785230303447486464#1:build-log.txt%3A6)3    1845 rules.go:145] validating repository order
I0430 08:52:21.054434    1845 rules.go:168] validating go versions
I0430 08:52:21.054601    1845 validate.go:36] validating directories exist in the kubernetes branch
E0430 08:52:21.633096    1845 main.go:45] Error : cri-client not found in branch master
F0430 08:52:21.633[7](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/123797/pull-publishing-bot-validate/1785230303447486464#1:build-log.txt%3A7)57    1845 main.go:47] Invalid rules file "/home/prow/go/src/k8s.io/kubernetes/staging/publishing/rules.yaml"
```

Or `pull-kubernetes-verify` fails when the code is missing:

```
Traceback (most recent call last):
  File "/home/prow/go/src/k8s.io/kubernetes/hack/make-rules/../../hack/verify-publishing-bot.py", line 134, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/prow/go/src/k8s.io/kubernetes/hack/make-rules/../../hack/verify-publishing-bot.py", line 129, in main
    raise Exception("missing rules for %s" % ','.join(str(s) for s in items))
Exception: missing rules for cri-client

```

Reply:
yep, we let `pull-publishing-bot-validate` fail for now as it is not a mandatory CI job and fix it immediately after